src/script.js

```diff
@@ -0,0 +1,388 @@
+  generateBoard();
+  guessCount++
+  totalGuesses.innerHTML = `Number of guesses: ${guessCount}`
+  messageWrapper.style.display = "block";
+}
+
+const alertMessage = (keepCharacter) => {
+  const { category, value } = currentQuestion;
+  
+  if (category === 'accessories') {
+    if (keepCharacter) {
+      alertMessageText.innerHTML = `Yes, the person wears ${value}! Keep all people that wear ${value}.`;
+    } else {
+      alertMessageText.innerHTML = `No, the person doesn't wear ${value}! Remove all people that wear ${value}.`;
+    }
+  } else if (category === 'other') {
+    if (keepCharacter) {
+      alertMessageText.innerHTML = `Yes, the person is a ${value}! Keep all people that are ${value}s.`;
+    } else {
+      alertMessageText.innerHTML = `No, the person is not a ${value}! Remove all people that are ${value}s.`;
+    }
+  } else {
+    if (keepCharacter) {
+      alertMessageText.innerHTML = 
+        `Yes, the person has ${value} ${category}! Keep all people with ${value} ${category}.`;
+    } else {
+      alertMessageText.innerHTML = 
+        `No, the person doesnt have ${value} ${category}! Remove all people with ${value} ${category}.`;
+    }
+  }
+  generateBoard()
+}
+
+const guess = (personToConfirm) => {
+  checkMyGuess(personToConfirm)
+}
+
+// If you confirm, this function is invoked
+const checkMyGuess = (personToCheck) => {
+  if (secretCharacter.name === personToCheck) {
+    winOrLoseWrapper.style.display = "flex";
+    winOrLoseText.innerHTML = `<span class="emoji"> 🥳 </span> Great Job! It was ${secretCharacter.name}.`
+    gameBoard.innerHTML = ``
+    questionSection.innerHTML = ""
+  } else {
+    winOrLoseWrapper.style.display = "flex";
+    winOrLoseText.innerHTML = `<span class="emoji"> 😓 </span> It is not ${personToCheck}. It was ${secretCharacter.name}.`
+    gameBoard.innerHTML = ``
+    questionSection.innerHTML = ""
+  }
+}
+
+const startTimer = () => {
+  clearInterval(timerInterval);
+
+  let seconds = 0; minutes = 0; hours = 0;
+
+    // Set a interval every 1000 ms
+    timerInterval = setInterval(() => {
+      timer.innerHTML =
+        "Time elapsed: " +
+        (hours ? hours + ':' : '') +
+        (minutes < 10 ? '0' + minutes : minutes) +
+        ':' +
+        (seconds < 10 ? '0' + seconds : seconds);
+
+      // Then, we add a second since one second has passed
+      seconds++;
+
+      // if seconds equals 60 seconds reset seconds to 0 and add a minute
+      if (seconds == 60) {
+        minutes++;
+        seconds = 0;
+      }
+
+      // if minutes equals 60 minutes reset minutes to 0 and add 1 hour
+      if (minutes == 60) {
+        hours++;
+        minutes = 0;
+      }
+    }, 1000);
+};
+
+// Invokes the start function when website is loaded
+startGame()
+selectQuestion()
+
+// All the event listeners
+restartButton.addEventListener("click", startGame);
+playAgainButton.addEventListener("click", () => window.location.reload());
```

Comment:
Smart how you added "window.location.reload", as this will also properly reload the page rather than only sending you back to the start page